packages/algorand-js/src/runtime/account.ts

```diff
@@ -98,4 +99,44 @@ export class StoreAccountImpl implements StoreAccount {
       appId: appId
     });
   }
+
+  createApp (appId: number, params: SSCDeploymentFlags): CreatedApp {
+    if (this.createdApps.length === 10) {
+      throw new Error('Maximum created applications for an account is 10');
+    }
+
+    const appParams: SSCParams = {
```

Comment:
Why do we call it `SSCParams`? How about callling it `SSCAttributes`?

Reply:
renamed